inception-project/inception
# #1069 - RefactorUncertaintySamplingStrategy

**What's in the PR**
* Refactor UncertaintySamplingStrategy.java.
* Removed code not related to Sampling Strategy from UncertaintySamplingStrategy.java
* Added this code to ActiveLearningSidebar.java
  * The generateNextSuggestion() method in this class contains a lot of code that is not directly relevant to generating the next set of candidate suggestions based on AL strategy. This code should be moved to ActiveLearningSidebar.java from where generateNextSuggestion() method is called.

**How to test manually**
* Start inception server and run the AL mode. There should be no change in the user experience.

**Automatic testing**
* [ ] PR does not include Automated Tests

**Documentation**
* [ ] PR does not update documentation
